rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ApiSpecTest.java

```diff
@@ -81,6 +82,7 @@ public void should_document_error_bad_request() throws Exception {
                                 fieldWithPath("message").description("The error message, e.g. JSON parse error: Unexpected end-of-input: expected close marker for Object"))));
     }
 
+    @Ignore
```

Comment:
Please add a description why this test is ignored

Reply:
Added description/comment why this test is currently ignored.